gravitee-am-ui/src/app/domain/settings/password-policy/domain-password-policy.component.ts

```diff
@@ -140,6 +145,31 @@ export class DomainPasswordPolicyComponent implements OnInit {
     this.passwordPolicy = { ...policy };
   }
 
+  private retrieveLinkedIdentityProviders(refresh: boolean) {
```

Comment:
question: There is a retrieveLinkedIdentityProviders with parameter refresh. Based on this name I would expect to retrieve providers from backend when refresh is set to true. In this method you fetch providers from DB when refresh is false, and use existing providers when refresh is set to true. Was that your intention?

Reply:
my intension is to rebuild the list of linkdIdp whitout calling the backend as I already have the list of all providers.
I will rename it to `listLinkedIdentityProviders(areProviderLoaded: boolean)`